.github/pull_request_template.md

```diff
@@ -0,0 +1,59 @@
+<!--
+
+--- PR Hygiene Checklist ---
+
+1. Make sure the changeset can be reviewed, keep it's scope and size succinct 
+2. Make sure your branch is from your fork and has a meaningful name
+3. Update the PR title: `BLUEBUTTON-99999 Add Awesomeness`
+4. Edit the text below - do not leave placeholders in the text.
+4.1. Remove sections that you don't feel apply
+5. Add any other details that will be helpful for the reviewers: details description, screenshots, etc
+6. Request a review from someone/multiple someones
+7. <optional> Review your changes yourself and write up any comments / concerns as if you were reviewing someone else's code.
+-->
+
+### Change Details
+
+<!-- Add detailed discussion of changes here: -->
+<!-- This is likely a summary, or the complete contents, of your commit messages -->
+
+### Acceptance Validation
+
+<!-- What should reviewers look for to determine completeness -->
+
+<!-- Insert screenshots if applicable (drag images here) -->
+
```

Comment:
I'm not clear on what is asked for in this section.

Reply:
I'm not either. I suppose it's intended to be any context the reviewer might need in order to determine correctness? I suppose it would look allot like the content of the JIRA ticket.

Would changing it to something like `Purpose of change` make more sense? I'd like it to be as clear as possible to a new contributor what to put in the PR description.